en/skills/11. Tables.ipynb

```diff
@@ -0,0 +1,458 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Exporting a CSV"
+      ],
+      "metadata": {
+        "id": "LtqgsVRMfLvL"
+      },
+      "id": "LtqgsVRMfLvL"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Exporting from Excel"
+      ],
+      "metadata": {
+        "id": "OXBo4HBy6w9k"
+      },
+      "id": "OXBo4HBy6w9k"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "\n",
+        "- Open your Excel file.\n",
+        "- Click on File in the top-left corner.\n",
+        "- Select Save As.\n",
+        "- Choose a location to save the file.\n",
+        "- In the Save as type dropdown, select CSV (Comma delimited) (*.csv).\n",
+        "- Click Save.\n",
+        "\n",
+        "Note:\n",
+        "- A warning may pop up, stating that some features of the workbook may be lost if saved in CSV format. Click Yes to confirm.\n"
+      ],
+      "metadata": {
+        "id": "KI3_vdMe6n9n"
+      },
+      "id": "KI3_vdMe6n9n"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Exporting from Google Sheets"
+      ],
+      "metadata": {
+        "id": "LSpApMWZ6yL_"
+      },
+      "id": "LSpApMWZ6yL_"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "- Open your Google Sheets file.\n",
+        "- Go to File in the menu bar.\n",
+        "- Hover over Download.\n",
+        "- Select Comma-separated values (.csv).\n",
+        "- The file will automatically download to your default download location."
+      ],
+      "metadata": {
+        "id": "HQP1W-s060ya"
+      },
+      "id": "HQP1W-s060ya"
+    },
+    {
+      "cell_type": "markdown",
+      "id": "fec8ab9f",
+      "metadata": {
+        "id": "fec8ab9f"
+      },
+      "source": [
+        "## Reading and Writing CSV Files\n",
+        "Just like reading from a CSV file, you can also write new data into a CSV file using Python’s `csv` module.\n",
+        "\n",
+        "#### Example (Writing):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "e491bbfc",
+      "metadata": {
+        "id": "e491bbfc"
+      },
+      "outputs": [],
+      "source": [
+        "import csv\n",
+        "\n",
+        "data = [[\"Nawatl\", \"English\"],\n",
+        "        [\"Wehka\", \"Far\"],\n",
+        "        [\"Oksepa\", \"Again\"],\n",
+        "        [\"Xinichchia\", \"Wait for me\"]]\n",
+        "\n",
+        "with open('data.csv', 'w', newline='') as file:\n",
+        "    writer = csv.writer(file)\n",
+        "    writer.writerows(data)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Once this file is created, it will appear wherever we set the path to. Since we didn't (we just named it \"output.csv\" it's directly in the same directory where we have this same file we're currently working with."
```

Comment:
Same question as above: how does this tie to ELT applications? As much as possible, we try to laser focused on the end products, since there's a ton of python/NLP content that we could potentially cover.

Reply:
Same response, but based on the fact that I do a lot of this for my work as there's usually not an ELT application and I have to work with the files themselves. But again, I can add that at a later point, I'll chat about it with Alexis.